src/utilities/eicrecon/print_info.h

```diff
@@ -49,4 +48,3 @@ void printJANAHeaderIMG() {
                  " YMMMM9 _dM_     _dMM_M_      \\M _dM_     _dMM_MMMMMMMM " << std::endl << std::endl;
 }
 
```

Comment:
Would be nice to not leave a trailing whitespace

Reply:
Yeah, but my sed fu only goes so far and I was going to have pre-commit do that... I'll take a look later.

Reply:
Resolved by #464.